MathMystics (Feature/project setup)

lucy-sees opened the pull request:
In this part of the project I did the following,

- Installed linters.
- Created my first React app.

DammyShittu reviewed (commented):
Hi @lucy-sees,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly fix your live demo or remove the section and header link on the readme.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

divinecharlotte reviewed (changes requested):
Hi @lucy-sees  , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- Clean code👍 
- Implemented some project requirements   👍 

 




 
![](https://media.giphy.com/media/tKyrc0NTlEcGWdThp4/giphy.gif)
## Required Changes ♻️

- [ ] kindly consider adding a working link under your LinkedIn account  and please do the same for the issue page under the contribution section to make your readme look more professional and more interactive  to the people visiting it


_Check the comments under the review._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

lucy-sees commented:
@divinecharlotte ,
Hi, my linkedIn profile link has been having an issue ever since I removed my linkedin account from hibernate. I've removed the link from my README file and reported to Linkedin customer support to help me fix the issue with that link. I will update the link back to my README file once it is resolved.

AbelG101 commented on README.md:
```diff
@@ -1,2 +1,162 @@
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
- [ ] [OPTIONAL] Hey @lucy-sees, the reason the issues page link is not working is because the relative path of  `../../issues/` is used it should normally work when the branch is merged but for your future reference you can use an absolute path like `https://github.com/lucy-sees/MathMystics/issues` to avoid such problems 😉.

lucy-sees replied:
Thank you, I will make sure I update that

AbelG101 reviewed (approved):
![](https://media.giphy.com/media/eKfovWZOlOpbTDEX9F/giphy.gif)

# Hi @lucy-sees 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Broken link has been removed ✔
- Correct use of Git Flow ✔
- All linter checks are passing ✔
- Descriptive PR title with a good summary ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
